docs/modules/user-guide/partials/con_using-top-level-attribute.adoc

```diff
@@ -0,0 +1,38 @@
+[id="con_using-top-level-attribute_{context}"]
+= Using top-level attribute
+
+[role="_abstract"]
+
+This section explains how to use top-level attribute in devfiles in order to reuse global attributes in different sections of the same devfile.
+
+.Devfile variable
+
+The devfile 2.0 specification includes global variables, which defines key-value pairs.
```

Comment:
Is it _only_ 2.0? Or is it 2.0 or higher? If so, use `2.x` to indicate any version of 2. Also, subject-verb agreement is off. See this rewrite:

`The devfile 2.x specification includes global variables which define key value pairs.`